homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -136,6 +113,16 @@ def unique_id(self) -> str:
         """Return a unique identifier for this entity."""
         return f"{self._station_id}_{self._fuel_type}"
 
+    @property
+    def device_info(self) -> DeviceInfo | None:
+        """Return device info."""
+        return DeviceInfo(
+            identifiers={(ATTR_ID, self._station_id)},
+            name=f"{self._brand} {self._street} {self._house_number}",
+            model=self._brand,
+            configuration_url="https://www.tankerkoenig.de",
+        )
```

Comment:
The device info should have entry type set to service.

https://developers.home-assistant.io/docs/device_registry_index#device-properties